.github/workflows/compatibility.yaml

```diff
@@ -0,0 +1,54 @@
+# This github workflow tests client and server compatibility for telepresence using nightly builds.
+#
+# See the "matrix" field below for which versions are tested.
+
+name: "Telepresence Test Matrix"
+on:
+  # Run daily at 9am UTC.
+  schedule:
+    - cron: '0 9 * * *'
+  # Run whenever this file is modified.
+  push:
+    paths:
+      - '.github/workflows/compatibility.yaml'
+
+jobs:
+  telepresence_matrix:
+    strategy:
+      fail-fast: false
+      matrix:
+        client_version: ["nightly"]
```

Comment:
Do we need to test server -> old client compatibility also? Or is one direction enough?

I guess that makes the most sense as a separate matrix 🤔

Reply:
If we want to test old client versions we can just extend the matrix to do that. My goal with the PR is mostly to get the mechanics in place.